apps/home/templates/apps_home/blocks/paragraph_block.html

```diff
@@ -0,0 +1,7 @@
+{% load block_tags %}
+
+<div class="modul-text_bild ds-block ds-block--body-bg">
+    <div class="servicepanel__main ds-block__paragraph">
+        {{ self.body }}
+    </div>
+</div>
```

Comment:
sorry I meant to use their markup, not just class name and sorry didn't add a link, not sure we should because it's public? anyway something like 
```
<section class="modul-text_bild">
    <div class="text textile">{{ self.body }}</div>
</section>
```

Reply:
Ah, cool, and sorry for merging! It looked fine in this PR, but not anymore with the headline anyway, so I will do that in that PR. @phillimorland